build.sbt

```diff
@@ -227,7 +226,7 @@ lazy val apiDocs = (project in file("api"))
   .enablePlugins(OpenApiGeneratorPlugin)
   .settings(
     name := s"$artifactNamePrefix-docs",
-
+    skipReleaseSettings,
```

Comment:
are all these explicit skipRelease settings needed?

Reply:
I was running into failures without having them specifically added, yeah.  It looked like it was still attempting some part of the publish procuess but had only partial config